.github/workflows/update_package_properties.yml

```diff
@@ -0,0 +1,52 @@
+name: Update Package Properties
+
+env:
+  GH_TOKEN: ${{ secrets.GH_TOKEN }}
+  PRJ_NAME: "citus"
+  TAG_NAME: ${{ github.event.inputs.tag_name }}
+  PRJ_VER: ${{ github.event.inputs.project_version }}
+  FANCY: ${{ github.event.inputs.fancy }}
+  FANCY_VERSION_NO: ${{ github.event.inputs.fancy_version_no }}
+  MICROSOFT_EMAIL: ${{ github.event.inputs.microsoft_email }}
+  NAME: ${{ github.event.inputs.name }}
+
+on:
+  workflow_dispatch:
+    inputs:
+      tag_name:
+        description: 'The tag name on the project repo to be released'
+      project_version:
```

Comment:
I think the project version should be derived from the tag name.

Reply:
@hanefi what is your opinion about it? Is there any exceptional case that breaks the relation between version and tag? If not I can remove the parameter version and correlate that varible with tag by removing 'v' prefix.

Reply:
I think it makes sense to do this. The package version and the tag name will always be the same (except for the `v` prefix)

Reply:
Fixed